com.regnosys.rosetta.tests/src/com/regnosys/rosetta/generator/java/expression/ListOperationTest.xtend

```diff
@@ -96,7 +96,7 @@ class ListOperationTest {
 						
 						protected List<Foo.FooBuilder> assignOutput(List<Foo.FooBuilder> filteredFoos, List<? extends Foo> foos) {
 							filteredFoos = toBuilder(MapperC.of(foos)
-								.filterItem(__item -> (Boolean)areEqual(__item.<Boolean>map("getInclude", _foo -> _foo.getInclude()), MapperS.of(Boolean.valueOf(true)), CardinalityOperator.All).get()).getMulti());
+								.filterItem(_item -> (Boolean)areEqual(_item.<Boolean>map("getInclude", __foo -> __foo.getInclude()), MapperS.of(Boolean.valueOf(true)), CardinalityOperator.All).get()).getMulti());
```

Comment:
I've improved our Java gen scoping mechanism: instead of always prefixing with __, I now add a number of _ depending on its nestedness.

Reply:
Nice